content/en/blog/2024/go-opentelemetry-io.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Planned Migration for go.opentelemetry.io
+linkTitle: Planned migration for go.opentelemetry.io
+date: 2024-08-23
+cSpell:ignore: Yahn
+author: >-
+  [Mike Dame](https://github.com/damemi) (Google),
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
+body_class: otel-with-contributions-from # TODO: remove this line if there are no secondary contributing authors
+issue: 5086
+sig: Go SIG
+---
+
+With contributions from Tyler Yahn and Austin Parker.
+
+## TL;DR
+
+The app that serves requests to [go.opentelemetry.io] will be migrating to a new
+host on September 5, 2024 at approximately 11:00AM Eastern Time. There are no
+changes required from you. We are making this announcement because it is
+possible, but unlikely, that there might be downtime during this transition.
+
+## Details
+
+OpenTelemetry is planning to migrate the application that serves all requests
+for `go.opentelemetry.io` to a new host project on September 5, 2024. On this
+date, the current DNS entries for `go.opentelemetry.io` will be updated to point
+to the new host.
+
+This should be a seamless transition, but there is the possibility of downtime
+during this time. The OpenTelemetry engineers will be actively monitoring the
+transition to minimize the risk.
+
+### Affected resources
+
+In the unlikely event of issues, the scope of these planned changes will affect
+Go package downloads for the following domains:
+
+- `go.opentelemetry.io/auto`
+- `go.opentelemetry.io/build-tools`
+- `go.opentelemetry.io/collector-contrib`
+- `go.opentelemetry.io/collector`
+- `go.opentelemetry.io/contrib`
+- `go.opentelemetry.io/otel`
+- `go.opentelemetry.io/proto`
+
+Note that this includes users of any of the following artifacts: the
+OpenTelemetry Go language libraries (including Contrib), the OpenTelemetry Go
+Auto-Instrumentation libraries, the OpenTelemetry Collector libraries (including
+Contrib), and the OpenTelemetry Collector Builder.
```

Comment:
How about making the "artifacts" list a bulleted list too? It would be easier to read that way.

Reply:
done